client/src/app/core/repositories/motions/motion-change-recommendation-repository.service.ts

```diff
@@ -136,15 +136,17 @@ export class MotionChangeRecommendationRepositoryService extends BaseRepositoryW
     ): ViewMotionChangeRecommendation {
```

Comment:
the return type is wrong. You can use `motionChangeRecommendation` since it is a plain object, but just not `ViewMotionChangeRecommendation`.